.github/buildomat/build-and-test.sh

```diff
@@ -95,9 +95,19 @@ ptime -m timeout 1h cargo test --doc --locked --verbose --no-fail-fast
 # Build the live-tests.  This is only supported on illumos.
 # We also can't actually run them here.  See the README for more details.
 if [[ $target_os == "illumos" ]]; then
+    banner "live-test"
     ptime -m cargo xtask live-tests
 fi
 
+# Do some test runs of the `ls-apis` command.
+banner ls-apis
+(
+    source ./tools/include/force-git-over-https.sh;
```

Comment:
I think what's essential here is the `export CARGO_NET_GIT_FETCH_WITH_CLI=true`.  Without this, the CI job fails because it can't access a token that can be used to clone the https URL.